src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -247,7 +247,7 @@ class Gold_Data(
                 ),
                 collections = emptyMap()
```

Comment:
Is this going to be populated as part of a subsequent PR?

Reply:
I could do that. I was trying to keep the size of the golden files down, but you're right it might be good to have an example checked in.